lib/firefox_profile.js

```diff
@@ -5,13 +5,13 @@
 
 var os = require('os'),
   path = require('path'),
+  util = require('util'),
```

Comment:
Note to self: This was added in nodejs v8.2, and v8 LTS cycle expired on December 31, 2019.